decidim/decidim
# Use UTC in the serializers for the date fields

#### :tophat: What? Why?

The CSV export in the answers for a questionnaire where always in UTC. This didn't respect how it works in other serializers. 

I searched for this same bug (using `to_s(:db)` and I could find a couple more places where this is being used, so those are fixed too, making them consistent with the rest of serializers (that I'm aware of at least). 

#### :pushpin: Related Issues
 
- Fixes #11200

#### Testing

1. Go to admin -> settings
2. Set Time Zone to `(GMT+0900) Tokyo`
3. Answer Survey
4. Go to admin -> the participatory processes -> survey
5. Click `Export all` and select CSV or Excel
6. Open exported file and See `Answered on` column
 

:hearts: Thank you!
